config/billing_rates.yml

```
@@ -92,6 +92,14 @@
 - { id: a3abe656-6ca2-4f17-a911-169c51a4a77a, resource_type: GitHubRunnerMinutes,     resource_family: standard-60-arm,     location: global,         unit_price: 0.0240000000, billed_by: amount,   active_from: 2023-01-01T00:00:00Z }
 - { id: 27d84309-e39d-8d78-b603-7632dafe374e, resource_type: GitHubRunnerMinutes,     resource_family: standard-gpu-6,      location: global,         unit_price: 0.0320000000, billed_by: amount,   active_from: 2023-01-01T00:00:00Z }
 - { id: c31b3640-92fa-49c3-bae8-c1011cc5f917, resource_type: GitHubRunnerConcurrency, resource_family: standard,            location: global,         unit_price: 0.0000248016, billed_by: duration, active_from: 2023-01-01T00:00:00Z }
+- { id: b1bf0809-d8b4-47b0-9a2a-c50f6037b4ca, resource_type: KubernetesCPVCpu,        resource_family: standard,            location: hetzner-fsn1,   unit_price: 0.0000000000, billed_by: duration, active_from: 2025-03-01T00:00:00Z }
```

IMHO, there's no need to use abbreviations. It makes less readable

```suggestion
- { id: b1bf0809-d8b4-47b0-9a2a-c50f6037b4ca, resource_type: KubernetesControlPlaneVCpu,        resource_family: standard,            location: hetzner-fsn1,   unit_price: 0.0000000000, billed_by: duration, active_from: 2025-03-01T00:00:00Z }
```

---

KubernetesControlPlaneVCpu is long, causing table spacing changes in the whole file, so I went with the short version. But since you and Burak liked it, I am making the change.